docs/content/posts/2025-03-03-release-kubernetes-operator-1.11.0.md

```diff
@@ -0,0 +1,90 @@
+---
+title:  "Apache Flink Kubernetes Operator 1.11.0 Release Announcement"
+date: "2025-03-03T18:00:00.000Z"
+authors:
+- gyfora:
+  name: "Gyula Fora"
+aliases:
+- /news/2025/03/03/release-kubernetes-operator-1.11.0.html
+---
+
+The Apache Flink community is excited to announce the release of Flink Kubernetes Operator 1.11.0!
+The version brings a number of important fixes and improvements to both core and autoscaler modules.
+
+We encourage you to [download the release](https://flink.apache.org/downloads.html) and share your experience with the
+community through the Flink [mailing lists](https://flink.apache.org/community.html#mailing-lists) or
+[JIRA](https://issues.apache.org/jira/browse/flink)! We're looking forward to your feedback!
+
+## Highlights
+
+### Flink 2.0 Preview Support
+
+The Flink Kubernetes Operator and Autoscaler 1.11.0 brings support for Flink 2.0 preview version. 
+This should help users to try out and verify the latest features in Flink planned for the 2.0 release.
+
+```yaml
+apiVersion: flink.apache.org/v1beta1
+kind: FlinkDeployment
+metadata:
+  name: basic-example
+spec:
+  image: flink:2.0
+  flinkVersion: v2_0
+...
+```
+
+Make sure to update the operator CRD to be able to deploy Flink 2.0 pipelines.
+
+***The support for Flink 2.0 in the operator and autoscaler is also in a preview state 
+and will be finalized after the official Flink 2.0.0 release***
+
+### Flink Operator Core
+
+#### More flexible Flink version based configs
+
+Default configs can now be defined for multiple versions at the same time using the following syntax:
+```properties
+# Set config for Flink 1.18 and larger
+kubernetes.operator.default-configuration.flink-version.v1_18+.key: value
+```
+
+#### Extended lifecycle states
+
+Two new lifecycle states (`DELETING` and `DELETED`) have been added to ensure that the operator correctly tracks the entire
+lifecycle of applications.
+
+This is especially useful when implementing a custom resource listener.
+
+#### Bug fixes for batch applications
+
+The version contains a number of important fixes for managing batch applications to avoid some annoying errors.
+
+### Autoscaler improvements
+
+The autoscaler module received a number of important tweaks, improvements and bugfixes. 
+
+Most notably:
+ - More robust scale down logic to reduce fluctuations [FLINK-36535](https://issues.apache.org/jira/browse/FLINK-36535)
+ - `autoscalerResetNonce` in Kubernetes JobSpec allows easy reset of the autoscaler state and parallelism overrides
+ - Publish metrics during stabilization period to reduce metric gaps
```

Comment:
Do we need to mention FLINK-36836 here? It updated some common options, maybe it's worth bringing this to users' attention.

Reply:
very good point, let me add that